<launch>
  <node name="image_proc" pkg="labrom_sensors" type="image_proc_node" ns="camera">
    <param name="image_transport" value="compressed"/>
    <param name="encoding" value="rgb8"/>
  </node>
</launch>
